### DISCUSSION TITLE
TS5+ support

### DISCUSSION THREAD

**Discussion by BR0kEN-** [jordimarimon/ts-ast-parser#202] [General]
Hello @jordimarimon. I saw the TS5 support is already in the `main` however no release tag is available. Would it be possible to make one please?

**Comment by jordimarimon** [comment]
Hi @BR0kEN- !

Thanks for opening a discussion!

Sorry for the delay, the library is still in beta and between the version `0.2.0` and `0.3.0` there have been a lot of breaking changes. It's my fault because I made a few mistakes while designing the architecture of the library. 

There will be a lot of new functionalities that will arrive in the next few weeks and I needed to change a lot of things before hand.

I have released a new version with support for TypeScript 5.0 and 5.1.

Make sure to check the [CHANGELOG.md](https://github.com/jordimarimon/ts-ast-parser/blob/main/packages/core/CHANGELOG.md) to see what has changed.

If you have a hard time updating, with the amount of breaking changes, feel free to let me know in this discussion or opening a new issue. I will be happy to help in the migration!

Also It may be useful while updating to look at the new [API Reference](https://jordimarimon.github.io/ts-ast-parser/api-reference/) section in the documentation even though it's not finished yet.

Thanks!
